frontend/src/lib/components/LemonTag/LemonTag.scss

```diff
@@ -4,7 +4,6 @@
     background: var(--border);
     padding: 0.125rem 0.25rem;
     border-radius: var(--radius);
-    text-transform: uppercase;
```

Comment:
This broke the pre-existing 30 instances of `LemonTag`! They were all intended to be all-caps and now accidentally aren't. It's totally okay to change the styling of a component, but existing UI needs to be taken into account.